packages/calypso-stripe/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+	"compilerOptions": {
+		"target": "ES5",
+		"lib": [ "DOM", "ES2015", "ES2017" ],
```

Comment:
The two `ES2015` and `ES2017` could be replaced by one `ES2020`. It determines what type definitions are used for platform stuff like `Array` or `String`.

Reply:
Thanks! I used `ESNext` as that seems to be the result of the discussions in #47002.